manifest.go

```diff
@@ -139,6 +140,14 @@ func (m *Manifest) toRaw() rawManifest {
 	return raw
 }
 
+// IsEmpty - Checks if payload is empty
+func (m *Manifest) IsEmpty() bool {
+	if m.Ovr == nil && len(m.Ignores) == 0 && len(m.Dependencies) == 0 {
```

Comment:
Let's add a check for `m.Required` as well and we'll have hit all the fields in the manifest.